[2024-05-22T22:36:04.187+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-22T22:36:04.257+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dataset.load manual__2024-05-22T22:09:25.534916+00:00 [queued]>
[2024-05-22T22:36:04.282+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dataset.load manual__2024-05-22T22:09:25.534916+00:00 [queued]>
[2024-05-22T22:36:04.284+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-05-22T22:36:04.320+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-05-22 22:09:25.534916+00:00
[2024-05-22T22:36:04.331+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1037) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-22T22:36:04.333+0000] {standard_task_runner.py:63} INFO - Started process 1061 to run task
[2024-05-22T22:36:04.334+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dataset', 'load', 'manual__2024-05-22T22:09:25.534916+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dataset.py', '--cfg-path', '/tmp/tmp72rul1in']
[2024-05-22T22:36:04.339+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask load
[2024-05-22T22:36:04.415+0000] {task_command.py:426} INFO - Running <TaskInstance: kaggle_dataset.load manual__2024-05-22T22:09:25.534916+00:00 [running]> on host 3943db1ddae8
[2024-05-22T22:36:04.544+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dataset' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-05-22T22:09:25.534916+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-22T22:09:25.534916+00:00'
[2024-05-22T22:36:04.546+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-22T22:36:04.562+0000] {logging_mixin.py:188} INFO - Hola Mundo
[2024-05-22T22:36:04.562+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-22T22:36:04.563+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-22T22:36:04.577+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kaggle_dataset, task_id=load, run_id=manual__2024-05-22T22:09:25.534916+00:00, execution_date=20240522T220925, start_date=20240522T223604, end_date=20240522T223604
[2024-05-22T22:36:04.631+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-22T22:36:04.656+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22T22:36:04.659+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
